lang/java/reef-bridge-client/src/main/java/org/apache/reef/bridge/client/YarnJobSubmissionClient.java

```diff
@@ -172,16 +172,13 @@ private void writeDriverHttpEndPoint(final File driverFolder,
     String trackingUri = null;
     LOG.log(Level.INFO, "Attempt to reading " + httpEndpointPath.toString());
     for (int i = 0; i < 60; i++) {
-      try {
-        LOG.log(Level.FINE, "Attempt " + i + " reading " + httpEndpointPath.toString());
-        if (fs.exists(httpEndpointPath)) {
-          final FSDataInputStream input = fs.open(httpEndpointPath);
-          final BufferedReader reader = new BufferedReader(new InputStreamReader(input, StandardCharsets.UTF_8));
-          trackingUri = reader.readLine();
```

Comment:
Hi! 

Overall, this PR looks good to me. But, did you test 'YarnJobSubmissionClient.java' on YARN environment? I think `readLine` might throw `IOException` although `httpEndpointPath` file exists.
The purpose of this for-loop is waiting the actual content of file is written completely, isn't it?

What about just changing `ex` into `ignored` instead and keep the original `try-catch` block? I think the main scope of this PR is coding-style instead of actual code logic.